nemo/collections/tts/models/audio_codec.py

```diff
@@ -474,9 +478,20 @@ def validation_step(self, batch, batch_idx):
         self.log_dict(metrics, on_epoch=True, sync_dist=True)
 
     @staticmethod
-    def _setup_train_dataloader(cfg):
+    def get_dataset(cfg):
+        with open_dict(cfg):
+            is_sharded = cfg.dataset.pop('is_sharded', False)
+
+        if is_sharded:
+            cfg.dataset._target_ = 'nemo.collections.tts.data.vocoder_dataset.TarredVocoderDataset'
+
         dataset = instantiate(cfg.dataset)
```

Comment:
For `TarredVocoderDataset` don't we need to pass the trainer `world size` and `global rank` into `instantiate`?